book/tables/ADA/adat01.qmd

```diff
@@ -7,20 +7,18 @@ subtitle: Baseline Prevalence and Incidence of Treatment Emergent ADA
 
 {{< include ../../_utils/envir_hook.qmd >}}
 
-:::: {.panel-tabset}
```

Comment:
hey Abi
Can you please check if this is working for you? I think that removing `:` from here (and also on the bottom) messes up the structure. This could be also my local stuff so please double check it on your machine

feature branch locally:
`quarto render tables/ADA/adat01.qmd`
![image](https://github.com/insightsengineering/tlg-catalog/assets/12943682/6e1d9a3a-ec13-446c-9363-eeecc6a9426a)

rendered:
https://insightsengineering.github.io/tlg-catalog/stable/tables/ADA/adat01.html
![image](https://github.com/insightsengineering/tlg-catalog/assets/12943682/c0abda10-5ac9-4c23-a30f-cc16325defe9)

Reply:
No worries I'll revert the changes to the styling and update the PR